tests/subsys/debug/coredump_backends/boards/yd_esp32.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&flash0 {
```

Comment:
Did you consider adding the `mcuboot` and `slot0,1` partitions as other ESP32 boards does, in order to support MCUboot?

Reply:
This board, like other ESP32 boards, is adding the `mcuboot` and `slot0,1` partitions via the `.dts` file located in the board's directory:

https://github.com/zephyrproject-rtos/zephyr/blob/935facc38c8340832ba1ba3516349c5f26a4a3bc/boards/xtensa/yd_esp32/yd_esp32.dts#L161-L197


In the `tests/subsys/debug/coredump_backends/boards/` dir, the `yd_esp32.overlay` file contains the same content as the Overlays used by other ESP32 boards:

- Checksum (SHA256):

![Screenshot01_Zephyr_2023-08-05](https://github.com/zephyrproject-rtos/zephyr/assets/103332557/02b8f3b2-d351-4863-b973-608ee1c682a6)

Reply:
yes sure, I didn't realize it is the `.overlay` file